### PR TITLE
add docs for dpop support coming in 6.3.0

### PR DESCRIPTION
**Pull request by brockallen**

### CODE REVIEW

**Comment by brockallen** [comment]
@josephdecock I'd love at some point if you had time to review and make any edits/changes for readability, clarity, etc.

File: IdentityServer/v6/docs/content/tokens/pop/dpop.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+The mechanism by which the client proves control of the private key (both when connecting to the token server and when calling an API) is by sending an additional JWT called a proof token on HTTP requests.
+This proof token is passed via the *DPoP* request header and contains the public portion of the JWK, and is signed by the corresponding private key.
+
+The creation and management of this DPoP key is up to the policy of the client.
+For example is can be dynamically created when the client starts up, and can be periodically rotated.
+The main constraint is that it must be stored for as long as the client uses any access tokens (and possibly refresh tokens) that they are bound to.
+
+#### Enabling DPoP in IdentityServer
+
+DPoP is something a client can use dynamically with no configuration in IdentityServer, but you can configure it as required.
```
Comment (josephdecock) [inline IdentityServer/v6/docs/content/tokens/pop/dpop.md:34]: I think this section needs a bit of clarification. I think what we're getting at is that you can always send a DPoP proof token to get a dpop token, but to require DPoP, you set the RequireDPoP flag? Is that right?

Comment (josephdecock) [reply IdentityServer/v6/docs/content/tokens/pop/dpop.md:34]: And is there a configuration that would disable it entirely?

Comment (brockallen) [reply IdentityServer/v6/docs/content/tokens/pop/dpop.md:34]: The intent of the statement was to say that you don't need to mark the client as required for dpop. The client can just add the dpop request values without the IS config requiring it. IOW, the client can just do what the client(s) samples do, and DPoP will work. The IS config for require dpop just says that the client must use dpop.

Comment (josephdecock) [reply IdentityServer/v6/docs/content/tokens/pop/dpop.md:34]: Ok, this doesn't need to hold up the pr, we can come back to it when/if we do the disable flag.

**Review by josephdecock — APPROVED** [review]

**Pull request merged**